tests/testsuite/git_commit.rs

```diff
@@ -0,0 +1,42 @@
+use ansi_term::Color;
+use std::process::Command;
+use std::{io, str};
+
+use crate::common::{self, TestCommand};
+
+#[test]
+fn test_render_commit_hash() -> io::Result<()> {
```

Comment:
Looks like a good test 👍 
I think the only remaining test would be another one testing `commit_hash_length` configuration.

Reply:
test added